Makefile

```diff
@@ -49,7 +49,7 @@ deps-list:
 	clojure -X:deps list
 
 repl:
-	clj -M:test -e "(require 'blaze.dev) (in-ns 'blaze.dev)" --repl
+	clj -M:test:repl/interactive -e "(require 'blaze.dev) (in-ns 'blaze.dev)"
```

Comment:
The original intention of the `-e` parameter was to automate the usage of the custom `blaze.dev` namespace. However, it does not run now... (I tried both with and without the `--repl` at the end).
Another option would be to switch to the standard `user` namespace, instead of the custom `blaze.dev` namespace.